application/sitebuilder/build.py

```diff
@@ -373,6 +416,22 @@ def build_other_static_pages(build_dir):
             write_html(file_path, content)
 
 
+def build_error_pages(build_dir):
+    templates_path = "application/templates"
+    relative_error_pages_path = "static_site/error"
+    full_error_pages_path = os.path.join(templates_path, relative_error_pages_path)
+
+    output_dir = os.path.join(build_dir)
+    os.makedirs(output_dir, exist_ok=True)
+
+    for error_page_path in glob.glob(os.path.join(full_error_pages_path, "*.html")):
+        source_file_path = re.sub(f"^{templates_path}{os.path.sep}", "", error_page_path)
```

Comment:
This took me a min to understand. Could we avoid this step by making the paths consistent?

Reply:
I did this in a weird way. I've simplified it and added some comments. Is it any better?

I want to leave the S3 structure as it is right now, because this ticket is more about just updating the tickets than it is about addressing the tech debt we have around the specific structure. Fixing that would be a larger piece of work, I think.